widgets/common/src/deployModal/DeployModalActions.tsx

```diff
@@ -0,0 +1,92 @@
+import { FunctionComponent } from 'react';
+
+const t = Stage.Utils.getT('widgets.common.deployments.deployModal');
+
+interface ApproveButtonsProps {
+    loading: boolean;
+    showDeployButton: boolean;
+    onInstall: () => void;
+    onDeploy: () => void;
+}
+
+const ApproveButtons: FunctionComponent<ApproveButtonsProps> = ({ showDeployButton, onInstall, loading, onDeploy }) => {
+    enum Buttons {
+        install,
+        deploy
+    }
+    const { ApproveButton, Button, Dropdown } = Stage.Basic;
+    const { useInput } = Stage.Hooks;
+    const [selectedButton, setSelectedButton] = useInput(Buttons.install);
+
+    if (!showDeployButton) {
+        <ApproveButton
+            onClick={onInstall}
+            disabled={loading}
+            content={t('buttons.deployAndInstall')}
+            icon="cogs"
+            className="green"
+        />;
+    }
+    return (
+        <Button.Group color="green">
+            {selectedButton === Buttons.install ? (
+                <ApproveButton onClick={onInstall} disabled={loading} content={t('buttons.install')} icon="cogs" />
+            ) : (
+                <ApproveButton onClick={onDeploy} disabled={loading} content={t('buttons.deploy')} icon="rocket" />
+            )}
+
+            <Dropdown
+                className="button icon down"
```

Comment:
I assume that you are using those classes here to get the expected style?

I managed to simplify it to:
```suggestion
                button
                className="icon"
```
which seems still not fully correct, but a bit more proper.

Reply:
I changed it: 4486b07ae139f30fb41fe85f2c7b81ea5da82906